components/css/mixins/_all.scss

```diff
@@ -1,4 +1,5 @@
 @import 'backgrounds';
 @import 'calculations';
 @import 'colors';
-@import 'fonts';
\ No newline at end of file
+@import 'fonts';
+@import 'h';
```

Comment:
Can we rename this `heading` or `headings`? I see you already have a partial named that, but I think this works since it's in a separate folder.

Reply:
Yes